src/js/views/patients/widgets/widgets.js

```diff
@@ -64,14 +64,21 @@ const widgets = {
     },
     template: hbs`{{formatMessage (intlGet "patients.widgets.widgets.sex") sex=sex}}`,
   },
-  status: {
+  status: View.extend({
     template: hbs`<span class="widgets__status-{{ status }}">{{formatMessage (intlGet "patients.widgets.widgets.status") status=status}}</span>`,
+    initialize() {
+      const workspacePatient = Radio.request('entities', 'get:workspacePatients:model', this.model.get('id'));
```

Comment:
never have to do `.get('id')` `.id` is always better.  But instead can this call `this.model.getWorkspacePatient()`?  We can also probably just `this.listenTo(workspacePatient, 'change:status', this.render)`

Reply:
I updated the listener to this in the status widget:

```js
initialize() {
  this.workspacePatient = this.model.getWorkspacePatient();

  this.listenTo(this.workspacePatient, 'change:status', this.render);
},
templateContext() {
  return {
    status: this.workspacePatient.get('status'),
  };
},
```